deployment/bicep/iiot-app.bicep

```diff
@@ -57,16 +57,33 @@ param applicationName string
 ])
 param location string = 'westeurope'
 
+@description('The AKS service principal object id')
+param aksObjectId string
+
+@description('Whether to create Azure Container registry with Service Principal role assignment')
+param acrCreate bool = false
+
 var applicationNameWithoutDashes = replace(applicationName, '-', '')
 var resourceGroupName = '${applicationName}-App'
 var storageAccountName = 'st${take(toLower(applicationNameWithoutDashes),22)}'
 var eventHubNameSpaceName = 'evh${take(toLower(applicationNameWithoutDashes),14)}'
+var acrName = 'acr${applicationNameWithoutDashes}'
```

Comment:
Not related to this PR, at some point, we should run a check on name creations and constraints for various resources we create.